### PR TITLE
Made env variables consistent in naming: Alex

### LINKED ISSUE #192: Standardise ENV variable names across scripts
As a developer I want to have my code run across multiple developers machines and across folders, so that the process of working together is more streamlined

- Go through all code and standardise ENV variable naming

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #192 

What I have done:
- Updated variable names
- Updated function names

### CODE REVIEW

File: weekly-openmeteo/load.py [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 
 def load_data(data: list[list]) -> None:
     '''Inserts data into a postgres database'''
     query = '''INSERT INTO solar_feature (county_id,sunrise_timestamp,sunset_timestamp)
-    VALUES (%s,%s,%s)'''
+    VALUES %s'''
     with get_connection() as conn:
         with conn.cursor() as cur:
-            cur.executemany(query, data)
+            execute_values(cur,query, data)
```
Comment (CalebMathers) [inline weekly-openmeteo/load.py:13]: Good switch to make it more efficient with execute values

File: aurorawatch-pipeline/README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ aws configure
 This will require sensitive information to be entered through the command line.
 
 5. Add to your `.env` file the following:
-    - REGION - region of your ECR repository
+    - AWS_REGION - region of your ECR repository
```
Comment (CalebMathers) [inline aurorawatch-pipeline/README.md:39]: Good change to keep in naming conventions with the terraform

File: dashboard/deploy.sh [hunk 3]
```diff
@@ -1,5 +1,5 @@
 source .env
-aws ecr get-login-password --region $REGION| docker login --username AWS --password-stdin $ECR_URI
+aws ecr get-login-password --region $AWS_REGION| docker login --username AWS --password-stdin $ECR_URI
```
Comment (CalebMathers) [inline dashboard/deploy.sh:2]: Good change to keep in naming conventions with the terraform

File: sms-checker/README.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ pip install -r requirements.txt
     - DB_PORT - port to access RDS (typically 5432 for postgres)
     - AWS_ACCESS_KEY - your personal access key to AWS
     - AWS_SECRET_KEY - your personal secret key to AWS
-    - REGION - the region of AWS that you will be using for this project
+    - AWS_REGION - the region of AWS that you will be using for this project
```
Comment (CalebMathers) [inline sms-checker/README.md:23]: Good change to keep in naming conventions with the terraform

File: sms-checker/message.py [hunk 5]
```diff
@@ -60,7 +60,7 @@ def get_client(service: str) -> client:
     '''Returns a AWS client'''
     return client(service, aws_access_key_id=ENV["MY_AWS_ACCESS_KEY"],
                         aws_secret_access_key=ENV["MY_AWS_SECRET_KEY"],
-                        region_name=ENV['REGION'])
+                        region_name=ENV['AWS_REGION'])
```
Comment (CalebMathers) [inline sms-checker/message.py:63]: Good change to keep in naming conventions with the terraform

File: weekly-openmeteo/README.md [hunk 6]
```diff
@@ -36,7 +36,7 @@ aws configure
 This will require sensitive information to be entered through the command line
 
 5. Add to your `.env` file the following:
-    - REGION - region of your ECR repository
+    - AWS_REGION - region of your ECR repository
```
Comment (CalebMathers) [inline weekly-openmeteo/README.md:39]: Good change to keep in naming conventions with the terraform

**Review by CalebMathers — APPROVED** [review]

File: sms-checker/README.md (hunk 4 above)
Comment (olliejth) [inline sms-checker/README.md:23]: Nice changes as discussed.

File: aurorawatch-pipeline/README.md (hunk 2 above)
Comment (olliejth) [reply aurorawatch-pipeline/README.md:39]: Good change to keep in naming conventions with the terraform

**Review by olliejth — APPROVED** [review]
thanks for making these changes.

File: weekly-openmeteo/load.py (hunk 1 above)
Comment (MurasatoNaoya) [reply weekly-openmeteo/load.py]: execute_values essentially allows bulk insertion of flattened values, much more efficient that iterative insertion. Important addition for efficiency.

File: aurorawatch-pipeline/README.md (hunk 2 above)
Comment (MurasatoNaoya) [reply aurorawatch-pipeline/README.md:39]: AWS_REGION is the tfvars env variable not REGION, LGTM.

File: sms-checker/message.py (hunk 5 above)
Comment (MurasatoNaoya) [reply sms-checker/message.py:63]: Good change to keep in naming conventions with the terraform

File: weekly-openmeteo/README.md (hunk 6 above)
Comment (MurasatoNaoya) [reply weekly-openmeteo/README.md:39]: Good change to keep in naming conventions with the terraform

**Review by MurasatoNaoya — COMMENTED** [review]
LGTM.

**Pull request merged**